src/data_utils/create_dataset/create_dataset.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import yaml
+import logging
+import pickle
+import os
+import multiprocessing
+import shutil
+from src.common.depthmap_toolkit import utils
+import sys
+from pathlib import Path
+sys.path.append('.')
+sys.path.append('../../common/depthmap_toolkit')
+sys.path.append(str(Path(__file__).parents[1]))
+# import pandas as pd
+from data_utils import CollectQrcodes  # noqa: E402
+from src.common.depthmap_toolkit import pcd2depth  # noqa: E402
+
+# Load the yaml file
+with open("src/data_utils/create_dataset/parameters.yml", "r") as ymlfile:
+    cfg = yaml.load(ymlfile, Loader=yaml.FullLoader)
+
+# Parse all the configuration variables
+
+db_file = cfg["database"]['db_connection_file']
+number_of_scans = cfg['scans']['scan_amount']
+calibration_file = cfg['calibration']['calibration_file']
+scangroup = cfg['data']['scangroup']
+target_folder = cfg['paths']['target_path']
+source = cfg['paths']['source_path']
+
+pcd_path = target_folder + 'pointclouds'
+if not os.path.exists(pcd_path):
+    os.makedirs(pcd_path)
+
+depthmap_path = target_folder + 'depthmaps'
+if not os.path.exists(depthmap_path):
+    os.makedirs(depthmap_path)
+
+rgb_path = target_folder + 'rgb'
+if not os.path.exists(rgb_path):
+    os.makedirs(rgb_path)
+
+dataset = CollectQrcodes(db_file)
+logging.info("Starting the dataset Preparation:")
+data = dataset.get_all_data()
+scangroup_data = dataset.get_scangroup_data(data=data, scangroup=scangroup)
+scangroup_qrcodes = dataset.get_unique_qrcode(scangroup_data)
+new_scangroup_data = dataset.get_usable_data(dataframe=scangroup_qrcodes, amount=number_of_scans, scan_group=scangroup)
+
+# new_scangroup_data = new_scangroup_data[new_scangroup_data['tag'] !=
+# 'delete']  #TODO: check for delete tag presence and remove those rows
+
+full_dataset = dataset.merge_qrcode_dataset(new_scangroup_data, scangroup_data)
+logging.info("Saving the csv file for EDA notebook.")
+full_dataset.to_csv('evaluation.csv', index=False)
+# full_dataset = pd.read_csv('src/data_utils/create_dataset/evaluation.csv')
+
+# Create the RGB csv file for posenet.
+
+get_posenet_data = dataset.get_posenet_results()
+get_rgb_artifacts = dataset.get_artifacts()
+
+get_rgb_qrcodedata = dataset.merge_data_artifacts(full_dataset, get_rgb_artifacts)
+get_posenet_results = dataset.merge_data_posenet(full_dataset, get_posenet_data)
+
+get_posenet_results.to_csv("RGB_poseresults.csv", index=False)
+# get_posenet_results = pd.read_csv('src/data_utils/create_dataset/RGB_pose1.csv')
+
+#Read the Calibration file and set the required shape fro height and width
+# calibration = utils.parseCalibration(calibration_file)
+Width = utils.setWidth(int(240 * 0.75))
```

Comment:
Global variables should be uppercase

```suggestion
WIDTH = utils.setWidth(int(240 * 0.75))
```